CHANGELOG.md

```diff
@@ -1,3 +1,112 @@
+# 1.2.0
+
+1. Breaking change which mostly maintains backwards compatibility, see
+   "Breaking Changes" below.
+2. Support for monadic generators e.g. [mwc-random](https://hackage.haskell.org/package/mwc-random).
+3. Monadic adapters for pure generators (providing a uniform monadic
+   interface to pure and monadic generators).
+4. Faster in all cases except one by more than x18 (N.B. x18 not 18%) and
```

Comment:
> Could benchmarks be submitted as a separate PR? At the moment it is difficult to verify your results, simply because new benchmarks were added in course of this very PR.

Good idea! See https://github.com/haskell/random/pull/64.